tasks/sync-granule/schemas/config.json

```diff
@@ -87,6 +87,12 @@
         "name": { "type": "string" },
         "path": { "type": "string" }
       }
+    },
+    "onDuplicateFilename": {
+      "type": "string",
+      "description": "Specifies how duplicate filenames should be handled",
```

Comment:
I think we should put a little more in the description. I think it's mostly self-explanatory, but I would question does `error` fail the entire task or just log an error somewhere and what does `version` do?

Reply:
OK. My bad. I wasn't sure how the config here even gets used, so I tried to leave my fuller explanations for the README. I'll update